cmd/manager/main.go

```diff
@@ -181,7 +180,7 @@ func main() {
 				// addon framework independently verifies the config-policy-controller via its lease
 				// see https://github.com/open-cluster-management/backlog/issues/11508
 				lease.CheckAddonPodFunc(generatedClient.CoreV1(), operatorNs, "app=policy-config-policy"),
-			)
+			).WithHubLeaseConfig(hubCfg, namespace)
```

Comment:
`namespace` is from WATCH_NAMESPACE, which is the cluster name: https://github.com/stolostron/klusterlet-addon-operator/blob/release-2.4/helm-charts/policy/templates/policy_framework_deployment.yaml#L102